jupyter/scripts/multi_step_learning/parameter_ranges/job_submitter.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+
+import argparse
+import json
+import multiprocessing
+import os
+import subprocess
+
+from jinja2 import Environment, FileSystemLoader
+from pathlib import Path
+
+def init_task_spooler():
```

Comment:
This comment seems to have got lost: Maybe factor out common code with `incremental_training/job_submitter`.

Reply:
Moved to `incremental_training/job_submitter.py`.